pkg/controller/node/nodecontroller.go

```diff
@@ -193,12 +202,32 @@ func NewNodeController(
 	return nc
 }
 
+// generateNewAvailabeCIDRs does generating new Available CIDRs index and insert
+// them into availableCIDRs set. Everytime it will generate 256 new CIDRs,
+// once there are no more CIDRs in the network, return false
+func (nc *NodeController) generateNewAvailabeCIDRs() bool {
+	if nc.firstAvailable+256 > nc.maxCIDRs {
+		glog.V(4).Infof("CIDR is not enough")
```

Comment:
"Out of CIDRs" is probably a better wording. Plus I think it should be a Warning.